src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAICitizenWander.java

```diff
@@ -45,17 +45,18 @@ public boolean shouldExecute()
         {
             return false;
         }
-        Vec3d Vec3d = RandomPositionGenerator.findRandomTarget(citizen, 10, 7);
-        if (Vec3d == null)
+
+        Vec3d vec3d = RandomPositionGenerator.getLandPos(citizen, 10, 7);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 7 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)